src/consumer/multi.rs

```diff
@@ -0,0 +1,338 @@
+use std::{
+    collections::{BTreeMap, VecDeque},
+    fmt::Debug,
+    future::Future,
+    iter,
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+use async_std::prelude::Stream;
```

Comment:
This causes a compile error when the `async_std` feature is not active.

Reply:
We can use `futures::Stream` which is runtime agnostic. I filed an issue: https://github.com/streamnative/pulsar-rs/pull/250.